history/titles/00_k_durotan.txt

```diff
@@ -10,6 +10,9 @@
 			}
 		}
 	}
+	611.1.12={holder=garrosh} #Garrosh
+	613.12.10={holder=68001} #Vol'jin
+	615.1.30={holder=42076} #Sylvanas
```

Comment:
Actually, Varok stayed in Orgrimmar instead of Sylvanas there as a leader of the orcs.

Reply:
Fixed in a73fedcf459b5b9f6d7920599cfffd5b7949cfe3.

Gave Durotan titles (`k_durotan`, `c_orgrimmar`, `c_rocktusk`, and `c_bladefist`) to Varok after Vol'jin dies, instead of Sylvanas.